tools/cloud-build/provision/README.md

```diff
@@ -0,0 +1,46 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Here it would be nice that you described a little what this does, even just your PR description would be ok.

Reply:
Similar documentation exist in https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/c209082b46fb327cdf7d9f2afdbebe36ac601269/tools/cloud-build, which should be updated.